internal/clientconn/conn.go

```diff
@@ -53,31 +53,34 @@ var AllModes = []Mode{NormalMode, ProxyMode, DiffNormalMode, DiffProxyMode}
 
 // conn represents client connection.
 type conn struct {
-	netConn net.Conn
-	mode    Mode
-	l       *zap.SugaredLogger
-	h       *pg.Handler
-	proxy   *proxy.Handler
+	netConn       net.Conn
+	mode          Mode
+	l             *zap.SugaredLogger
+	h             common.Handler
+	m             *ConnMetrics
+	proxy         *proxy.Handler
```

Comment:
It always bothered me that `proxy.Handler` does not implement Handler interface. But now we have a better name – router. Let's rename that type

Reply:
renamed.